ts/features/wallet/onboarding/cobadge/screens/start/CoBadgeChosenBankScreen.tsx

```diff
@@ -46,25 +78,44 @@ const CoBadgeChosenBankScreen = (props: Props): React.ReactElement => {
   const abiInfo: Abi | undefined = props.abiList.find(
     aI => aI.abi === props.abi
   );
-  return (
-    <SearchStartScreen
-      testID={props.testID}
-      methodType={"cobadge"}
-      onCancel={props.onCancel}
-      onSearch={props.searchAccounts}
-      handlePartecipatingBanksModal={openPartecipatingBankModal}
-      handleTosModal={openTosModal}
-      bankName={abiInfo?.name}
-    />
+
+  if (
+    abiInfo === undefined &&
+    props.abi !== undefined &&
+    isReady(props.abiListStatus)
+  ) {
+    props.onGeneralError("Abi not found in abilist in CoBadgeChosenBankScreen");
+    return null;
+  }
+
+  return fold(
+    props.abiListStatus,
+    () => abiListLoadingError(false, props.onCancel, props.loadAbiList),
+    () => abiListLoadingError(true, props.onCancel, props.loadAbiList),
+    _ => (
+      <SearchStartScreen
+        testID={props.testID}
+        methodType={"cobadge"}
+        onCancel={props.onCancel}
+        onSearch={props.searchAccounts}
+        handlePartecipatingBanksModal={openPartecipatingBankModal}
```

Comment:
```suggestion
        handleParticipatingBanksModal={openParticipatingBankModal}
```

Reply:
Fixed in [66cac06](https://github.com/pagopa/io-app/pull/2940/commits/66cac063ac2a2dd6a21a9e9287d258e426da1e1f)